crates/re_viewer/src/web.rs

```diff
@@ -73,12 +73,18 @@ impl WebHandle {
     }
 
     #[wasm_bindgen]
-    pub fn toggle_panel_overrides(&self) {
+    pub fn toggle_panel_overrides(&self, value: Option<bool>) {
```

Comment:
I find the existence of this API as a toggle (as opposed to just something like `set_panel_overrides_active`) to be a bit unexpected.

I'm glad to see us turning it into a hybrid set/toggle, though the name as toggle still seems like it might hurt discoverability / expectation somewhat.